src/components/dialogs/parameters/parameters.js

```diff
@@ -79,8 +79,18 @@ import {
 
 export const CloseButton = ({ hideParameters, classeStyleName }) => {
     return (
-        <Button onClick={hideParameters} className={classeStyleName}>
-            <FormattedMessage id="close" />
+        <LabelledButton
+            callback={hideParameters}
+            label={'close'}
+            name={classeStyleName}
+        />
+    );
+};
+
+export const LabelledButton = ({ callback, label, name }) => {
```

Comment:
antipattern to have such a small component
antipattern to rename props (onClick -> callback)